src/dso/cli/__init__.py

```diff
@@ -68,11 +69,18 @@ def dso_compile_config(all, args):
     default=bool(int(os.environ.get("DSO_SKIP_COMPILE", 0))),
     help="Do not compile configs before loading it. The same can be achieved by setting the `DSO_SKIP_COMPILE=1` env var.",
 )
+@click.option(
+    "--as_json",
```

Comment:
```suggestion
    "--json",
```

let's keep it simpler

Reply:
ah, you probably did this because there's a conflict with `import json`. 

One option is to use something like
```
from json import dump as dump_json
```

Reply:
yeah, but I agree. I will change it to json and modify it accordingly.